WordPress/WordPressUITests/Tests/MainNavigationTests.swift

```diff
@@ -17,26 +17,41 @@ class MainNavigationTests: XCTestCase {
         removeApp()
     }
 
-    func testTabBarNavigation() throws {
+    // We run into an issue where the People screen would crash short after loading.
+    // See https://github.com/wordpress-mobile/WordPress-iOS/issues/20112.
+    //
+    // It would be wise to add similar tests for each item in the menu (then remove this comment).
+    func testLoadsPeopleScreen() throws {
         XCTAssert(MySiteScreen.isLoaded(), "MySitesScreen screen isn't loaded.")
 
-        _ = try mySiteScreen
-            .tabBar.goToReaderScreen()
+        mySiteScreen.goToMenu()
+        mySiteScreen.goToPeople()
 
-        XCTAssert(ReaderScreen.isLoaded(), "Reader screen isn't loaded.")
+        // We don't have a ScreenObject for the people screen,
+        // so we just check that the previous screen is not loaded
+        XCTAssertFalse(MySiteScreen.isLoaded(), "MySitesScreen screen isn't loaded.")
+    }
```

Comment:
The diff got mangled, but this is the only real change: A new `testLoadsPeopleScreen()` method.

I did not touch the other one.